cornell-dti/cu-apts
# Tab and footer

### Summary 
This pull request implements showing a 404 page to the user if they navigate to a url for a landlord that doesn't exist, or a url for an apartment that doesn't exist. It also implements some small bug fixes.
- [x] Closes issue #106 
- [x] Closes issue #107 
- [x] Closes issue #121
- [x] Changed the styling for the privacy policy to make it look clickable by making it orange
- [x] Changed the wording in the footer from “Got a question that isn’t answered?” to "Want to get in touch?"
- [x] Changed the website favicon to the latest one
- [x] Changed the website tab title to "CUAPTS"
- [x] Implemented showing a 404 page instead of spinning forever if the user navigates to an invalid landlord or apartment id 
### Test Plan 
Run `yarn start` in the root folder. Check that the tab title, favicon, footer wording, and privacy policy is correct on the website.
- Try navigating to a landlord by clicking on their card from the home page. Confirm that the landlord page loads normally.
- Change the url to `landlord/landlord_id`, where `landlord_id` is a non-existent landlord. Confirm that the 404 page loads instead of the progress bar spinning forever.
